net/davix/src/TDavixFile.cxx

```diff
@@ -417,6 +417,17 @@ void TDavixFileInternal::parseConfig()
    if (gDebug > 0)
       Info("parseConfig", "Setting CAcheck to %s", ((ca_check_local) ? ("true") : ("false")));
 
+   std::string prefix = "Bearer ";
+   // setup Bearer token auth
+   env_var = gEnv->GetValue("Davix.HTTP.Token", getenv("HTTP_TOKEN"));
+      if (gDebug > 0)
+         Info("parseConfig", "Using Bearer TOKEN: %s", env_var);
```

Comment:
I think this breaks if `env_var` is `nullptr`

Reply:
I would suggest using `BEARER_TOKEN` instead of `HTTP_TOKEN`; the former is what `gfal2` uses.

Reply:
`TOKEN` is not an acronym.  For the debug messages, just use `token`.

This is also a security credential - I would strongly suggest to redact it from debug output lest these get copy/pasted into the root forum.